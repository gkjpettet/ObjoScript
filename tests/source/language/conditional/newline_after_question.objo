// Newline after '?'.
System.print(true ?
  "yes" : "no") // expect error.