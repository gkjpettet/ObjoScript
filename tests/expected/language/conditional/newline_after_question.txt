Expected an expression. Instead got `String`.